libs/application/api/core/src/lib/application/application.service.ts

```diff
@@ -128,8 +129,13 @@ export class ApplicationService {
         [Op.and]: [
           {
             [Op.or]: [
-              { applicant: nationalId },
-              { assignees: { [Op.contains]: [nationalId] } },
+              ...(actor
+                ? [
+                    { applicant: nationalId },
+                    { assignees: { [Op.contains]: [nationalId] } },
```

Comment:
assignees should probably be actors and checking if it contains the actor nationalId.. right?

Reply:
Oops!